### PR TITLE
Publish to Pypi with Travis.

### PR DESCRIPTION
**Pull request by benpatterson**

### CODE REVIEW

**Comment by benpatterson** [comment]
@jzoldak you could start provisional review. I have a ticket out to DevOps to get credentials.

File: .travis.yml [hunk 1]
```diff
@@ -21,3 +21,11 @@ script:
 after_success:
     - "pip install coveralls"
     - coveralls
+deploy:
+    provider: pypi
+    user: edx
+    password:
+        secure: foo
+    distributions: "sdist"
```
Comment (benpatterson) [inline .travis.yml]: this is what our bok-choy instructions specify. I was ruling out other distributions with this change (the example on the wiki also lists `bdist_wheel`).

**Comment by benpatterson** [comment]
@jzoldak Ready for your review now.

Thanks @fredsmith

**Comment by jzoldak** [comment]
:+1: :rocket:

**Pull request merged**